opba-facade-protocol-api-shared/src/main/java/de/adorsys/opba/protocol/api/services/scoped/validation/IgnoreBankValidationRuleDto.java

```diff
@@ -0,0 +1,15 @@
+package de.adorsys.opba.protocol.api.services.scoped.validation;
+
+import de.adorsys.opba.protocol.api.dto.codes.FieldCode;
+import lombok.Builder;
+import lombok.Value;
+
+@Builder
+@Value
+public class IgnoreBankValidationRuleDto {
```

Comment:
It is better to provide an interface that is wrapping entity - as there is no mapping except 1-1